test/test_no_prefix/faucet.config.js

```diff
@@ -0,0 +1,17 @@
+"use strict";
+let path = require("path");
+
+module.exports = {
+	sass: [{
+		source: "./src/index.scss",
+		target: "./dist/bundle.css",
+		browserslist: false
+	}],
+	manifest: {
+		file: "./dist/manifest.json",
+		value: f => `/assets/${f}`
+	},
```

Comment:
Why are you testing the manifest? Seems like an unnecessary maintenance burden for plugins (except perhaps perhaps for the combination of faucet-sass and faucet-static).